pkgs/applications/audio/vital/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
```

Comment:
```suggestion
, makeBinaryWrapper
```

Reply:
Interesting, is `makeBinaryWrapper` something I might be able to use on the plugins themselves (in `out/lib/clap/Vital.clap`, etc.)?

Reply:
Also, this change should probably be applied to the reaper package too.

Reply:
`makeBinaryWrapper` is just a replacement for `makeWrapper`.

Instead of creating a script, it will created a compiled binary. On Darwin system, `makeWrapper` creates some issues, while `makeBinaryWrapper` does not. This is the reason why I always advise people to use it by default.

Reply:
Oh cool! Good to know!